File: VB/GridDemo.Wpf/ModuleResources/PrintMasterDetailViewModel.vb
Imports System
Imports DevExpress.Mvvm.DataAnnotations
Imports DevExpress.Xpf.Printing

Namespace GridDemo
    <POCOViewModel>
    Public Class PrintMasterDetailViewModel
        Inherits PrintViewModelBase

        Protected Sub New()
        End Sub

        Public Overrides Sub OnLoaded(ByVal printableControl As IPrintableControl)
            MyBase.OnLoaded(printableControl)
            ShowPreviewInNewTab()
        End Sub

        Protected Overrides Function GetTitle() As String
            Return "Print Preview"
        End Function
    End Class
End Namespace

File: CS/GridDemo.Wpf/ModuleResources/PrintMasterDetailViewModel.cs
using System;
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Xpf.Printing;

namespace GridDemo {
    [POCOViewModel]
    public class PrintMasterDetailViewModel : PrintViewModelBase {

        protected PrintMasterDetailViewModel() { }

        public override void OnLoaded(IPrintableControl printableControl) {
            base.OnLoaded(printableControl);
            ShowPreviewInNewTab();
        }

        protected override string GetTitle() {
            return "Print Preview";
        }
    }
}
